atom/browser/api/atom_api_menu_views.cc

```diff
@@ -60,6 +60,8 @@ void MenuViews::PopupAt(Window* window, int x, int y, int positioning_item) {
 }
 
 void MenuViews::ClosePopupAt(int32_t window_id) {
+  if (menu_runners_[window_id])
+    menu_runners_[window_id]->Cancel();
```

Comment:
Correct me if I am wrong, doesn't the destruction of the menu runner call [Cancel](https://cs.chromium.org/chromium/src/ui/views/controls/menu/menu_runner_impl.cc?dr=CSs&l=49) eventually.

Reply:
I'm not saying you are wrong, but the test does fail if I remove this `Cancel` call, so I guess that even if it's called, the delegate is not informed about it.

Reply:
@zcbenz any thoughts on this ? I am not familiar with this code path.